types/nightwatch/test/testActions.ts

```diff
@@ -1,3 +1,7 @@
+import { NightwatchAPI } from "nightwatch";
+
+function isNightwatchAPI(v: NightwatchAPI) {}
```

Comment:
Can we move this to a common `util` file?

Reply:
Move all the common functions to `test/utils.ts`.